backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/lower/BridgesBuilding.kt

```diff
@@ -249,7 +249,13 @@ private fun Context.buildBridge(startOffset: Int, endOffset: Int,
             }
         }
 
-        +irReturn(delegatingCall)
+        if (bridge.returnType.isNullableNothing()
```

Comment:
What is this required for? Does `+irReturn(delegatingCall)` work in this case?

Reply:
Indeed it is not needed, I didn't expect `unbox` to correctly work on `null` :)